docs/en/manuals/google-play-instant.md

```diff
@@ -0,0 +1,118 @@
+
+To be able to publish your game as Google Play Instant app you need to set your project properly:
+
+1. Create custom `AndroidManifest.xml` file and add next two attributes to the `<manifest>` element:
+
+```lua
+xmlns:dist="http://schemas.android.com/apk/distribution"
+android:targetSandboxVersion="2"
+```
+How it should looks with default AndroidManifest.xml for example:
+
+```lua
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="{{android.package}}"
+        android:versionCode="{{android.version_code}}"
+        android:versionName="{{project.version}}"
+        android:installLocation="auto"
+        xmlns:dist="http://schemas.android.com/apk/distribution"
+        android:targetSandboxVersion="2">
+```
+
+2. Add a dependency to the Google Instant Apps extension in your “game.project” file. Add “https://github.com/AGulev/instantapp/archive/master.zip” or point to the ZIP file of a [specific release](https://github.com/AGulev/instantapp/releases) to the Dependencies property.
+
+![Project settings](images/gpi/game_project.png)
+
+3. Download libraries: Project->Fetch Libraries
+4. Bundle `apk` Project->Bundle->Android Application
+5. Create a `zip` archive with `apk` file
+6. Now you can upload this `zip` in your Google Play Console as Android Instant App
+
+### Version codes
+Pay attention to [the recommendations about the version codes](https://developer.android.com/topic/google-play-instant/getting-started/game-instant-app#version-codes): Instant Game version code needs to be less than version code of the installable game.
+
+![Project settings](images/gpi/version_code.png)
+
+### android:targetSandboxVersion="2"
+
+If you set `android:targetSandboxVersion="2"` in the main installable game you will be able to work with the same files as instant game (a save file for example). But then you would have restrictions in the main application. More information in [the official documentation](https://developer.android.com/guide/topics/manifest/manifest-element#targetSandboxVersion).
+::: sidenote
+Once an app is installed, you can only update its target sandbox value to a higher value. To downgrade the target sandbox value, you must uninstall the app and replace it with a version whose manifest contains a lower value for this attribute.
+::: 
+Even if you set different `android:targetSandboxVersion` in the installable game and instant game you still able to use `instantapp.set_cookie()` and `instantapp.get_cookie()` for communication between the game versions.
+
+
+## API Usage
+
+The Google Play Instant extension is accessible through the `instantapp.*` namespace where it wraps Java [PackageManagerCompat methods](https://developers.google.com/android/reference/com/google/android/gms/instantapps/PackageManagerCompat) in a Lua API.
+
+If you are working on cross-platform application the best practice to check the existence of `instantapp` module, this module exists only in android bundle:
```

Comment:
on cross-platform -> on a cross-platform
best practice to check -> best practice is to check
module, this module -> module since this module